docs/source/selection.rst

```diff
@@ -63,7 +63,13 @@ These selection methods can be modified to be semi-supervised by using the PCovR
 
     .. automethod:: select
 
-.. _ortho-api:
+Random Partitioning
```

Comment:
I would put it this in a separate section Model Selection, since its purpose is far away from CUR and FPS

Reply:
Done!